biggraphite/test_utils.py

```diff
@@ -187,6 +189,11 @@ def setUpClass(cls):
             cls.cassandra_helper.setUpClass()
             cls.ACCESSOR_SETTINGS.update(
                 cls.cassandra_helper.get_accessor_settings())
+        if "elasticsearch" in driver_name:
+            cls.elasticsearch_helper = ElasticsearchHelper()
+            cls.elasticsearch_helper.setUpClass()
+            cls.ACCESSOR_SETTINGS.update(
+                cls.elasticsearch_helper.get_accessor_settings())
```

Comment:
[remark] This part is very likely to fail with an hybrid accessor (see #370). It is not an issue right now but we have to keep it in mind for a short-term fix.